src/react-sortable-tree.js

```diff
@@ -603,6 +603,7 @@ class ReactSortableTree extends Component {
           toggleChildrenVisibility={this.toggleChildrenVisibility}
           {...sharedProps}
           {...nodeProps}
+          {...rowProps}
```

Comment:
What do you think about passing `rowProps` (or specific attributes such as `rowProps.isScrolling`) to `generateNodeProps` in line 573? Then, one could use these props from within `generateNodeProps` without having to define an own `nodeContentRenderer`.